CLI/src/main/java/com/CSCI32006/CLI/SetupDatabase.java

```diff
@@ -41,39 +41,30 @@ private void setDatabaseConn(String username, String password){
         jdbcTemplate = new JdbcTemplate(dataSource);
     }
 
-    private @NotNull StringInput setupContext(boolean mask, String name, String defaultValue) {
-        StringInput component = new StringInput(getTerminal(), name, defaultValue);
-        component.setResourceLoader(getResourceLoader());
-        component.setTemplateExecutor(getTemplateExecutor());
-        if(mask) {
-            component.setMaskCharacter('*');
-        }
-        return component;
-    }
-
     @Command(command = "quit", alias = "exit", description = "Exit the shell", interactionMode = INTERACTIVE, group = "Built-In Commands")
     public void quit() throws SQLException {
         logout();
         throw new ExitRequest();
     }
 
-    @Command(command = "logout", description = "Logs out of the database")
+    @Command(command = "logout database", description = "Logs out of the database")
     void logout() throws SQLException {
         if(jdbcTemplate != null){
             Objects.requireNonNull(jdbcTemplate.getDataSource()).getConnection().close();
             dataSource.getConnection().close();
             session.disconnect();
+            getTerminal().writer().println("Successfully disconnected!");
```

Comment:
I'm not seeing this message in terminal.

Reply:
I see this now, I didn't have the updated build like we discussed in discord, thought this was the same as the setup build.